index.html

```diff
@@ -46,46 +46,257 @@ <h3 class="subtitle">
+        </div>
+        <h3 class="project-title">Multi-Post stories</h3>
+        <p class="describe">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <div class="main-tags">
+          <div>
+            <a href="#"><img src="images/tags/Tag.png" alt="" /></a>
+          </div>
+          <div>
+            <a href="#"><img src="images/tags/Tag1.png" alt="" /></a>
+          </div>
+          <div>
+            <a href="#"><img src="images/tags/Tag2.png" alt="" /></a>
+          </div>
+          <div>
+            <a href="#"><img src="images/tags/Tag3.png" alt="" /></a>
+          </div>
+  
+        </div>
+          <img class= 'main-butt' src="images/Enable.png" alt="button">
+      </div>
+
+      <div class="cards">
+        <div class="card">
+            <div class="project-details">
+                <div class="project-content">
+                    <h2 class="project-name">Professional Art Printing Data</h2>
+                    <p class="description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                        has been the industry's standard
+                    </p>
+                    <ul class="boxes">
+                        <li><a href="#"><img src="images/tags/Tag5.png" alt="html"></a></li>
+                        <li><a href="#"><img src="images/tags/Tag6.png" alt="bootstrap"></a></li>
+                        <li><a href="#"><img src="images/tags/Tag7.png" alt="ruby"></a></li>
+                    </ul>
+                </div>
+            </div>
+            <a href="#" class="button see-project large">
+                See Project
+            </a>
+        </div>
       </div>
+        <div class="cards">
+          <div class="card">
+              <div class="project-details">
+                  <div class="project-content">
+                      <h2 class="project-name">Professional Art Printing Data</h2>
+                      <p class="description">
+                          A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                          has been the industry's standard
+                      </p>
+                      <ul class="boxes">
+                          <li><a href="#"><img src="images/tags/Tag5.png" alt="html"></a></li>
+                          <li><a href="#"><img src="images/tags/Tag6.png" alt="css"></a></li>
+                          <li><a href="#"><img src="images/tags/Tag7.png" alt="ruby"></a></li>
+                      </ul>
+                  </div>
+              </div>
+              <a href="#" class="button see-project large">
+                  See Project
+              </a>
+          </div>
+        </div>
```

Comment:
- [ ] Please do you best to follow this design on Figma and make your button look the same as on the  [figma template ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)